src/commands/contract/deploy.ts

```diff
@@ -68,9 +72,7 @@ export class DeployContract extends SwankyCommand<typeof DeployContract> {
       );
     }
 
-    if(!flags.account && this.swankyConfig.defaultAccount === null) {
-      throw new ConfigError("No default account set. Please set one or provide an account alias with --account");
-    }
+    ensureAccountIsSet(flags.account, this.swankyConfig);
 
     const accountAlias = flags.account ?? this.swankyConfig.defaultAccount;
```

Comment:
Display the expected `accountAlias` in "ConfigError("Provided account alias not found in swanky.config.json")"